pkg/ipamd/ipamd.go

```diff
@@ -198,13 +198,15 @@ type IPAMContext struct {
 	maxENI                    int
 	maxPrefixesPerENI         int
 	unmanagedENI              int
-	warmENITarget             int
-	warmIPTarget              int
-	minimumIPTarget           int
-	warmPrefixTarget          int
-	primaryIP                 map[string]string // primaryIP is a map from ENI ID to primary IP of that ENI
-	lastNodeIPPoolAction      time.Time
-	lastDecreaseIPPool        time.Time
+	nonZeroCardENI            int
```

Comment:
nit: for this to be extensible in the future, lets consider a field tracking the number of network cards on the instance